### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1093] [Galaxy Trivia] [answered]
```
302  202  102  002  991  ?
```

**Comment by peterbaccaro** [comment]
The next number is **891**.

This is because if you reverse each of the numbers you see that the numbers are counting down from 203 and the next number is 198 which becomes 891 when it is reversed.

**Comment by thuongtruong1009** [comment, marked as answer]
Looking closely, you can notice that the sequence is a mix of numbers and their mirror images, with the last digit of each number being flipped.

302 → 202 → 102 → 002 → 991

Following this pattern, the next number would be 191 (flipping the last digit of 991).
